tortoise/verifying_tortoise.go

```diff
@@ -48,13 +48,14 @@ type turtle struct {
 
 	Verified types.LayerID
 
-	// using the array to be able to iterate from latest elements easily.
-	BlocksToBlocks map[types.LayerID]map[types.BlockID]Opinion // records hdist, for each block, its votes about every previous block
+	// use 2D array to be able to iterate from latest elements easily
+	BlockOpinionsByLayer map[types.LayerID]map[types.BlockID]Opinion // records hdist, for each block, its votes about every
+	// previous block
 	// Use this to be able to lookup blocks Opinion without iterating the array.
 	BlocksToBlocksIndex map[types.BlockID]int
```

Comment:
this is un-used debt and should be removed as well. (I can do it in separate pr though)